nx/include/switch/services/capsa.h

```diff
@@ -0,0 +1,34 @@
+/**
+ * @file capsu.h
+ * @brief Album Accessor (caps:a) service IPC wrapper.
+ * @author Behemoth
+ * @copyright libnx Authors
+ */
+#pragma once
+#include "../types.h"
+#include "../sf/service.h"
+#include "../services/caps.h"
+
+typedef enum {
```

Comment:
Should be in caps.h (where's the name from)?

Reply:
just a guess